src/ecezo/ecezo.c

```diff
@@ -488,7 +488,7 @@ int ecezo_send_command(const ecezo_context dev, char *cmd, char *buffer,
     // our write buffer
     char writeBuffer[ECEZO_MAX_BUFFER_LEN];
 
-    strncpy(writeBuffer, cmd, ECEZO_MAX_BUFFER_LEN);
+    memcpy(writeBuffer, cmd, ECEZO_MAX_BUFFER_LEN-1);
     writeBuffer[ECEZO_MAX_BUFFER_LEN - 1] = 0;
```

Comment:
Seems redundant, you can initialize the entire buffer and remove this line.

Reply:
Not sure if that will be efficient.

Reply:
Nvm, after a second look I figured this terminates the buffer correctly when `cmd >= writeBuffer`.